src/stories/containers/users/delete-account/delete-account.tsx

```diff
@@ -14,57 +14,83 @@ import { toast } from 'react-toastify';
 import Notification, { ContainerNotification } from '../../../components/notification/notification';
 import CheckMark from '../../../components/svg/check-mark';
 import Warning from '../../../components/svg/warning';
+import { LOGIN_REQUEST } from '../../auth/login/login.api';
+import request from 'graphql-request';
+import { GRAPHQL_ENDPOINT } from '../../../../config/endpoints';
 
 export default () => {
-  const testingPassword = '1234';
   const router = useRouter();
-  const { clientRequest } = useAuthContext();
+  const { clientRequest, user } = useAuthContext();
   const { userName, id } = router.query;
   const [value, setValue] = useState('');
   const { isLight } = useThemeContext();
   const handleChange = useCallback((value: React.ChangeEvent<HTMLInputElement>) => {
     setValue(value.target.value);
   }, []);
 
+  const handleGoBack = useCallback(() => {
+    router.back();
+  }, [router]);
+
   const handleDeleteAccount = useCallback(async () => {
     try {
-      const { query: gqlQuery, filter } = USERS_DELETE_FROM_ADMIN(id as string);
-      const data = await clientRequest?.request(gqlQuery, filter);
-      if (data.userDelete) {
-        toast(
-          ({ closeToast }) => <Notification icon={<CheckMark />} borderColor="#B6EDE7" handleClose={closeToast} />,
-          {
-            position: toast.POSITION.BOTTOM_CENTER,
-            autoClose: 3000,
-            hideProgressBar: true,
-            closeButton: false,
-          }
-        );
-        setTimeout(() => {
-          router.push('/auth/manage');
-        }, 3000);
-      } else {
-        toast(
-          ({ closeToast }) => (
-            <Notification
-              icon={<Warning />}
-              borderColor="#FBE1D9"
-              handleClose={closeToast}
-              message="There was some problem deleting your account"
-            />
-          ),
-          {
-            position: toast.POSITION.BOTTOM_CENTER,
-            autoClose: 3000,
-            hideProgressBar: true,
-            closeButton: false,
-          }
-        );
+      const { query: gqlQueryLogin, input } = LOGIN_REQUEST(user?.username || '', value);
+      const response = await request(GRAPHQL_ENDPOINT, gqlQueryLogin, input);
+      if (response) {
+        const { query: gqlQuery, filter } = USERS_DELETE_FROM_ADMIN(id as string);
+        const data = await clientRequest?.request(gqlQuery, filter);
+        if (data.userDelete) {
+          toast(
```

Comment:
maybe we can create a `toast` helper function to trigger this one with the values so we can reuse it and avoid repeating things like color, positions, icons, etc. This will be very handy in this feature and future features that have this kind of toasts

Reply:
Already done